src/xscen/utils.py

```diff
@@ -962,7 +966,7 @@ def clean_up(  # noqa: C901
 
     See Also
     --------
-    xclim.core.calendar.convert_calendar
+    xclim.core.calendar.convert_calendar, xarray.DataArray.clip
```

Comment:
I'm not sure if base operations in xarray should be included in "See Also"? Otherwise, you might also want to add e.g. xr.DataArray.round, etc, and this never stops? I don't know, I also understand that 'clip' is less transparent than 'round', so maybe this justifies its inclusion. Anyways, feel free to ignore this nitpick

Reply:
I think it should include all function where xs.clean_up directly passes the args to the external function.